### DISCUSSION TITLE
I can't login because I didn't receive the verification code sent to my phone number

### DISCUSSION THREAD

**Discussion by TuanLe12** [vercel/community#2872] [Help] [answered]
### Summary

![image](https://github.com/vercel/community/assets/136550447/141811c6-f15e-4d62-b411-a6434392b208)

I had a situation where I was forced to verify by phone number but I didn't receive any code to fill in. Can anyone who has encountered this situation tell me how to handle this situation?

### Example

https://scontent.fhan3-2.fna.fbcdn.net/v/t1.15752-9/353074124_254213183924941_1681296581136796079_n.png?_nc_cat=107&ccb=1-7&_nc_sid=ae9488&_nc_ohc=9nrxw7a1llIAX-6H1Is&_nc_ht=scontent.fhan3-2.fna&oh=03_AdTHmR3DVYMUGilQhjr8BNTxIhODmxe_2bV3-9Piqu8RRw&oe=64B1BFED

### Steps to Reproduce

https://scontent.fhan3-2.fna.fbcdn.net/v/t1.15752-9/353074124_254213183924941_1681296581136796079_n.png?_nc_cat=107&ccb=1-7&_nc_sid=ae9488&_nc_ohc=9nrxw7a1llIAX-6H1Is&_nc_ht=scontent.fhan3-2.fna&oh=03_AdTHmR3DVYMUGilQhjr8BNTxIhODmxe_2bV3-9Piqu8RRw&oe=64B1BFED
did not receive the code for verification

**Comment by amyegan** [comment, marked as answer]
For registration and login issues, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team 🙏

**Reply by Saad5400** [reply]
I have to log in to submit a case .. and I'm unable to login due to the SMS not being delivered.

However I did contact both of registration@vercel.com and support@vercel.com.

It's been weeks without any reply.

Side note: I'm not the original issue creator.

**Reply by amyegan** [reply]
Sending that email is one of the ways you can open a case. If you can share the case number from either of those emails, I can check on the ticket status for you.

**Reply by Saad5400** [reply]
Thanks
00167455
00168709

**Reply by Saad5400** [reply]
Hey, so any news?
